docs/02-knora-ontologies/knora-base.md

```diff
@@ -766,83 +732,45 @@ the user has permission to see the source and target resources.
-
-`mappingHasXMLAttribute` (0-n)
-
-:   Maps XML attributes to standoff properties using
-`MappingXMLAttribute`. See below.
-
-`mappingHasStandoffDataTypeClass` (0-1)
-
-:   Indicates the standoff data type class of the standoff class the XML element is mapped to.
-
-`mappingElementRequiresSeparator` (1)
-
-:   Indicates if there should be an invisible word separator inserted after the XML element in the RDF-standoff
-representation. Once the markup is stripped, text segments that belonged to different elements may be concatenated.
+- `mappingHasXMLTagname` (1): The name of the XML element that is mapped to a standoff class.
+- `mappingHasXMLNamespace` (1): The XML namespace of the XML element that is mapped to a standoff class. If no namespace is given, `noNamespace` is used.
+- `mappingHasXMLClass` (1): The name of the class of the XML element. If it has no class, `noClass` is used.
+- `mappingHasStandoffClass` (1): The standoff class the XML element is mapped to.
+- `mappingHasXMLAttribute` (0-n): Maps XML attributes to standoff properties using `MappingXMLAttribute`. See below.
+- `mappingHasStandoffDataTypeClass` (0-1): Indicates the standoff data type class of the standoff class the XML element is mapped to.
+- `mappingElementRequiresSeparator` (1): Indicates if there should be an invisible word separator inserted after the XML element in the RDF-standoff representation.
+  Once the markup is stripped, text segments that belonged to different elements may be concatenated.
 
 A `MappingXMLAttribute` has the following properties:
+- `mappingHasXMLAttributename`: The name of the XML attribute that is mapped to a standoff property.
+- `mappingHasXMLNamespace`: The namespace of the XML attribute that is mapped to a standoff property. If no namespace is given, `noNamespace` is used.
+- `mappingHasStandoffProperty`: The standoff property the XML attribute is mapped to.
 
-`mappingHasXMLAttributename`
-
-:   The name of the XML attribute that is mapped to a standoff property.
-
-`mappingHasXMLNamespace`
-
-:   The namespace of the XML attribute that is mapped to a standoff property. If no namespace is given, `noNamespace` is
-used.
-
-`mappingHasStandoffProperty`
-
-:   The standoff property the XML attribute is mapped to.
-
-Knora includes a standard mapping used by the SALSAH GUI. It has the IRI
-`http://rdfh.ch/standoff/mappings/StandardMapping` and defines mappings for a few elements used to write texts with
-simple markup.
+DSP-API includes a standard mapping used by the DSP APP. It has the IRI `http://rdfh.ch/standoff/mappings/StandardMapping` and defines mappings for a few elements used to write texts with simple markup.
 
 #### Standoff in Digital Editions
 
-Knora's standoff is designed to make it possible to convert XML documents to standoff and back. One application for this
+DSP-API's standoff is designed to make it possible to convert XML documents to standoff and back. One application for this
```

Comment:
I think we should stick to one version `DSP API` which in most cases you've used or `DSP-API` as stands here.

Reply:
according to a quick search, `DSP-API` is used a lot more than `DSP API`, so I tried unifying to that. But I am sure I have missed some instances somewhere.